### PR TITLE
Helm Chart for the csi-secrets-store-provider-aws provider

### PR DESCRIPTION
**Pull request by pierluigilenoci**
### Issue

This PR is related to #633

### Description of changes

New Helm Chart for csi-secrets-store-provider-aws

### Checklist
- [x] Added/modified documentation as required (such as the `README.md` for modified charts)
- [x] Incremented the chart `version` in `Chart.yaml` for the modified chart(s)
- [x] Manually tested. Describe what testing was done in the testing section below
- [x] Make sure the title of the PR is a good description that can go into the release notes

### Testing

Test done here https://github.com/aws/secrets-store-csi-driver-provider-aws/pull/35

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by pierluigilenoci** [comment]
@bwagner5 let me know if there is anything else I can do. 

Fast recap: 
* https://github.com/aws/eks-charts/issues/633 (the issue related to this PR)
* https://github.com/aws/eks-charts/pull/651 (this PR)
* https://github.com/aws/eks-charts/issues/354 (a related issue in this repo)
* https://github.com/aws/secrets-store-csi-driver-provider-aws/issues/15 (an issue in the software repo)
* https://github.com/aws/secrets-store-csi-driver-provider-aws/pull/35 (PR in the software repo)
* https://github.com/aws/secrets-store-csi-driver-provider-aws/issues/55 (a related issue in the software repo)
* https://github.com/aws/secrets-store-csi-driver-provider-aws/pull/56 (related PR in the software repo)
* https://github.com/portefaix/portefaix-hub/pull/103 (related PR in another repo)

**Comment by bwagner5** [comment]
I'm testing now, so hopefully we can get this in today! :D

**Comment by bwagner5** [comment]
Tested helm chart and looks good! 

One final question: Is the tgz of the secrets-store-csi-driver necessary to check-in?

File: stable/csi-secrets-store-provider-aws/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+- [Helm3](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+### Installing the Chart
+
+- This chart installs the [secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) and the AWS Key Management Service provider for the driver
+
+```shell
+helm repo add eks https://aws.github.io/eks-charts
+helm install eks/csi-secrets-store-provider-aws --generate-name
```
[the remark this answers is not in the document]
Comment (pierluigilenoci) [reply stable/csi-secrets-store-provider-aws/README.md]: Done

File: stable/csi-secrets-store-provider-aws/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| `image.tag`| Image tag | `1.0.r2-2021.08.13.20.34-linux-amd64` |
+| `nodeSelector` | Node Selector for the daemonset on nodes | `{}` |
+| `tolerations` | Tolerations for the daemonset on nodes  | `[]` |
+| `ports` | Liveness and readyness tcp probe port  | `8989` |
+| `resources`| Resource limit for provider pods on nodes | `requests.cpu: 50m`<br>`requests.memory: 100Mi`<br>`limits.cpu: 50m`<br>`limits.memory: 100Mi` |
+| `podLabels`| Additional pod labels | `{}` |
+| `podAnnotations` | Additional pod annotations| `{}` |
+| `updateStrategy` | Configure a custom update strategy for the daemonset on nodes | `RollingUpdate`|
+| `rbac.install` | Install default service account | true |
+| `rbac.serviceAccount.name` | Service account to be used. If not set and serviceAccount.create is true a name is generated using the fullname template. | |
```
[the remark this answers is not in the document]
Comment (pierluigilenoci) [reply stable/csi-secrets-store-provider-aws/README.md:45]: added the entry + other entries that were missing

**Comment by pierluigilenoci** [comment]
> Tested helm chart and looks good!

👍🏻 🆒 🚀  

> One final question: Is the tgz of the secrets-store-csi-driver necessary to check-in?

It was uploaded by mistake. My bad.

**Comment by pierluigilenoci** [comment]
@bwagner5 thank you too!

**Pull request merged**